pkg/service/operation/storage/bolt.go

```diff
@@ -1,18 +1,40 @@
 package storage
 
 import (
+	"fmt"
 	"github.com/goccy/go-json"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/tbd54566975/ssi-service/internal/util"
 	"github.com/tbd54566975/ssi-service/pkg/storage"
 	"go.einride.tech/aip/filtering"
+	"strings"
 )
 
 const (
-	namespace = "operation"
+	namespace  = "operation"
+	submission = "submission"
 )
 
+const SubmissionParentResource = "/presentations/submissions"
+
+func namespaceFromID(id string) string {
+	i := strings.LastIndex(id, "/")
+	if i == -1 {
+		return ""
```

Comment:
We might want to panic here, we don't want to somehow write to the namespace ""

Reply:
I think this is ok because of https://github.com/boltdb/bolt/blob/master/bucket.go#L167

A panic would kill the server, so it's better to avoid it.